app/components/modal/edit-user/template.hbs

```diff
@@ -0,0 +1,64 @@
+<ModalDialog>
+  <div class='modal-dialog modal-xl'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <h5 class='modal-title'>Edit user: {{@user.username}}</h5>
+      </div>
+      <div class='modal-body'>
+        <div class='mb-3'>
+          <label for='emailUserInput' class='form-label'>Enter new email</label>
+          <input
+            type='e-mail'
+            class='form-control'
+            id='editEmailInput'
+            placeholder='{{@user.email}}'
+            value={{this.email}}
+            {{on 'input' this.onEmailChange}}
+          />
```

Comment:
How about to create shared input component in this PR? Which by default will have `type='text'` and `class='form-control'`. Example invoke in this case:
```hbs
<Shared::Input
  @id='editEmailInput'
  @type='e-mail'
  @placeholder={{@user.email}}
  @value={{this.email}}
  @onChange={{this.onEmailChange}}
/>
```